src/content/en/tools/lighthouse/audits/_shared/query.md

```diff
@@ -0,0 +1,15 @@
+Below the audit, Lighthouse displays something like `1 element fails
+this test`. The number varies depending on how many elements are failing.
+Click this label to expand the list. You can find each element in your
+DOM by running the [`$()`][qs], [`$$()`][qsa], and [`$x()`][xp] functions
+from the Chrome DevTools Console.
+
+You can also use [ChromeLens][ChromeLens] to find the elements. See
```

Comment:
I think this might work for the contrast example but I'm not sure if it works for every test we're doing. Does ChromeLens let you query for a specific element or does it just highlight elements that fail its own tests?